witness/golang/cmd/witness/impl/witness.go

```diff
@@ -55,23 +53,15 @@ type ServerOpts struct {
 	DBFile string
 	// The signer for the witness.
 	Signer note.Signer
-	// The file containing log configuration information.
-	ConfigFile string
+	// The log configuration information.
+	Config LogConfig
 }
 
-// loadConfig loads the log configuration information from a file and into a map.
-func loadConfig(configFile string) (map[string]witness.LogInfo, error) {
-	fileData, err := ioutil.ReadFile(configFile)
-	if err != nil {
-		return nil, fmt.Errorf("failed to read from config file: %v", err)
-	}
-	var js LogConfig
-	if err := json.Unmarshal(fileData, &js); err != nil {
-		return nil, fmt.Errorf("failed to parse config file as proper JSON: %v", err)
-	}
+// loadConfig loads the log configuration information into a map.
+func loadConfig(config LogConfig) (map[string]witness.LogInfo, error) {
```

Comment:
Maybe we could rename this to be something like `buildLogMap` or similar?

Reply:
Sure, done.